modules/add_book.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable import/prefer-default-export */
+export const addBook = (booklist, save, load) => {
```

Comment:
- [ ] I suggest that you should not disable any `eslint` guidelines set for this project because `eslint` check clean code and javascript best practices usage :blush: I think `addBook` is the only function you are exporting in this file so eslint is recommending for you to use the `export default` :relaxed: